app/addons/activetasks/assets/less/activetasks.less

```diff
@@ -63,3 +64,148 @@
   }
 }
 
+.no-matching-database-on-search {
+  color: #e33f3b;
+}
+
+p.multiline-active-tasks-message {
+  margin: 0;
+
+  &.time:nth-child(2) {
+    color: #888;
+  }
+}
+
+#dashboard-upper-content {
+  padding-right: 20px;
+}
+
+.active-tasks.dashboard-upper-menu {
+  left: 220px;
+
+  .closeMenu & {
+    left: 64px;
+  }
+}
+
+.dashboard-lower-menu {
+  padding-top: 90px;
+  padding-left: 20px;
+}
+
+input[type="text"].searchbox {
+  width: 200px;
+  height: 40px;
+  float: right;
+  margin:0px;
+}
+
+#toggle-filter-tab:hover {
+  color: white;
+  background-color: #e33f3b;
+}
+
+.filter-tray.toggleFilterTray-enter {     /* starting css*/
+  overflow: hidden;
+  padding-top: 0px;
+  max-height: 0px;
+}
+                                          /* animate opening */
+.filter-tray.toggleFilterTray-enter.toggleFilterTray-enter-active {     
+  max-height: 300px;
+  height: auto;
+  transition: max-height 0.5s linear;
+}
+
+.filter-tray {                             /* final css */
+  height: auto;
+  max-height: 300px;                      
+  overflow: hidden;
+}
+                                          /* animate closing */
+.filter-tray.toggleFilterTray-leave.toggleFilterTray-leave-active {     
```

Comment:
tip: if you change the 4 comments to // they don't get rendered into the generated css